docker/Dockerfile.dev

```diff
@@ -0,0 +1,11 @@
+FROM node:14-alpine
```

Comment:
Try using a slim image instead.

Reply:
> Try using a slim image instead.

Apart of the size and the distro, what's the difference between alpine and slim?

Reply:
Well, for development has sense to use slim because being a debian distro there are chances to install any additional package with apt, as for production I prefer to keep it with alpine since the smaller the image the better